src/CowEvcClosePositionWrapper.sol

```diff
@@ -0,0 +1,277 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity ^0.8;
+
+import {IEVC} from "evc/EthereumVaultConnector.sol";
+
+import {ICowSettlement, CowWrapper} from "./CowWrapper.sol";
+import {IERC4626, IBorrowing, IERC20} from "euler-vault-kit/src/EVault/IEVault.sol";
+import {SafeERC20Lib} from "euler-vault-kit/src/EVault/shared/lib/SafeERC20Lib.sol";
+import {CowEvcBaseWrapper} from "./CowEvcBaseWrapper.sol";
+
+/// @title CowEvcClosePositionWrapper
+/// @notice A specialized wrapper for closing leveraged positions with EVC
+/// @dev This wrapper hardcodes the EVC operations needed to close a position:
+///      1. Execute settlement to acquire repayment assets
+///      2. Repay debt and return remaining assets to user
+/// @dev The settle call by this order should be performing the necessary swap
+/// from collateralVault -> IERC20(borrowVault.asset()). The recipient of the
+/// swap should *THIS* contract so that it can repay on behalf of the owner. Furthermore,
```

Comment:
I suppose it's a leftover, but this comment is very incorrect, I see later that the target should be the owner. In general, this section should be updated.